### DISCUSSION TITLE
Set the temporary file location

### DISCUSSION THREAD

**Discussion by maopei820219** [dlemstra/Magick.NET#1462] [General] [answered]
The default location for temporary file is %SystemRoot%\Temp, for instance C:\Windows\Temp. Is there a way to configure it to an alternate directory?

**Comment by dlemstra** [comment, marked as answer]
You can change the temporary directory that ImageMagick uses with `MagickNET.SetTempDirectory`.

**Reply by maopei820219** [reply]
Thank you very much, Dirk!
